test/NavSpec.js

```diff
@@ -151,139 +151,139 @@ describe('<Nav>', () => {
-
-    it('should not allow focusing on disabled tabs', () => {
-      const links = wrapper.find('a').map((n) => n.getDOMNode());
-
-      expect(links[0].getAttribute('tabindex')).to.not.equal('-1');
-      expect(links[1].getAttribute('tabindex')).to.equal('-1');
-      expect(links[2].getAttribute('tabindex')).to.not.equal('-1');
-      expect(links[3].getAttribute('tabindex')).to.equal('-1');
-      expect(links[4].getAttribute('tabindex')).to.not.equal('-1');
-    });
-
-    it('should focus the next tab on arrow key', () => {
-      const anchors = wrapper.find('a');
-
-      anchors
-        .at(0)
-        .tap((a) => a.getDOMNode().focus())
-        .simulate('keydown', {
-          key: 'ArrowRight',
-        });
-
-      expect(wrapper.prop('activeKey')).to.equal('3');
-
-      expect(document.activeElement).to.equal(anchors.at(2).getDOMNode());
-    });
-
-    it('should focus the previous tab on arrow key', () => {
-      wrapper.setProps({ activeKey: 5 });
-
-      const anchors = wrapper.find('a');
-
-      anchors
-        .at(4)
-        .tap((a) => a.getDOMNode().focus())
-        .simulate('keydown', {
-          key: 'ArrowLeft',
-        });
-
-      expect(wrapper.prop('activeKey')).to.equal('3');
-      expect(document.activeElement).to.equal(anchors.at(2).getDOMNode());
-    });
-
-    it('should wrap to the next tab on arrow key', () => {
-      wrapper.setProps({ activeKey: 5 });
-      const anchors = wrapper.find('a');
-
-      anchors
-        .at(4)
-        .tap((a) => a.getDOMNode().focus())
-        .simulate('keydown', {
-          key: 'ArrowDown',
-        });
-
-      expect(wrapper.prop('activeKey')).to.equal('1');
-      expect(document.activeElement).to.equal(anchors.at(0).getDOMNode());
-    });
-
-    it('should wrap to the previous tab on arrow key', () => {
-      const anchors = wrapper.find('a');
-
-      anchors
-        .at(0)
-        .tap((a) => a.getDOMNode().focus())
-        .simulate('keydown', {
-          key: 'ArrowUp',
-        });
-
-      expect(wrapper.prop('activeKey')).to.equal('5');
-      expect(document.activeElement).to.equal(anchors.at(4).getDOMNode());
-    });
-
-    it('should forward to a onKeyDown listener', () => {
-      const anchors = wrapper.find('a');
-
-      anchors
-        .at(0)
-        .tap((a) => a.getDOMNode().focus())
-        .simulate('keydown', {
-          key: 'ArrowUp',
-        });
-
-      sinon.assert.calledOnce(keyDownSpy);
-    });
-
-    ['a', 'b', 'left', 'up'].forEach(({ key }) => {
-      it(`should ignore non-arrow keys: case ${key}`, () => {
-        const anchors = wrapper.find('a');
-
-        anchors
-          .at(0)
-          .tap((a) => a.getDOMNode().focus())
-          .simulate('keydown', {
-            key,
-          });
-
-        expect(document.activeElement).to.equal(anchors.at(0).getDOMNode());
-        sinon.assert.calledOnce(keyDownSpy);
-      });
-    });
-  });
+  // describe('keyboard navigation', () => {
```

Comment:
Currently this doesn't work because restart ui checks for existence of a tab context before handling keys.  I guess this might be ok?  If users want keyboard control, they can use `Tabs` instead.

Reply:
I think this is correct yeah. At least i can't imagine why you'd want this keyboard behavior outside of a tablist